### PR TITLE
Add presentation and publication information (issue #21)

### PR DESCRIPTION
**Pull request by fermadeiral**

### CODE REVIEW

File: cfp.html [hunk 1]
```diff
@@ -193,6 +193,17 @@ <h2>Submission Guidelines</h2>
                                 In case you have any doubt about the preparation of your manuscript following the double-blind rules, do not hesitate to send an email to <a href="mailto:TODO">TODO</a>.
                             </p>
                         </div>
+
+                        </br>
+
+                        <h2>Presentation and Publication</h2>
+                        <div class="description-one">
+                            <p align="justify">
+                                Upon notification of acceptance, authors of accepted papers will be asked to prepare their camera-ready versions.
```
Comment (felipeebert) [inline cfp.html:202]: "the" authors

File: cfp.html [hunk 2]
```diff
@@ -193,6 +193,17 @@ <h2>Submission Guidelines</h2>
...
+                                At least one author of each accepted paper is expected to register and present the research at the VEM 2021 workshop.
```
Comment (felipeebert) [inline cfp.html:203]: present the "paper"?

**Review by felipeebert — COMMENTED** [review]
Some minor comments.

**Comment by fermadeiral** [comment]
@felipeebert ready for review again.

**Pull request merged**